### PR TITLE
Add a Delete link from all languages in homepage edition

### PR DESCRIPTION
**Pull request by jojo787898**

### CODE REVIEW

File: main/admin/configure_homepage.php [hunk 1]
```diff
@@ -1207,9 +1242,10 @@ class="form-control"><?php echo $notice_text; ?></textarea>
                         if (!empty($enreg)) {
                             $edit_link = ' <a href="'.$selfUrl.'?action=edit_tabs&amp;link_index='.$tab_counter.'" ><span>'.Display::return_icon('edit.png', get_lang('Edit')).'</span></a>';
                             $delete_link = ' <a href="'.$selfUrl.'?action=delete_tabs&amp;link_index='.$tab_counter.'"  onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete')).'</span></a>';
+                            $delete_all = ' <a href="'.$selfUrl.'?action=delete_all&amp;link_index='.$tab_counter.'"  onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete All')).'</span></a>';
```
Comment (jmontoyaa) [inline main/admin/configure_homepage.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 329 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260&bid=17216516)

File: main/admin/configure_homepage.php [hunk 2]
```diff
@@ -1207,9 +1242,10 @@ class="form-control"><?php echo $notice_text; ?></textarea>
...
                             $tab_string = str_replace(
                                 ['href="'.api_get_path(WEB_PATH).'index.php?include=', '</li>'],
-                                ['href="'.api_get_path(WEB_CODE_PATH).'admin/'.basename($selfUrl).'?action=open_link&link=', $edit_link.$delete_link.'</li>'],
+                                ['href="'.api_get_path(WEB_CODE_PATH).'admin/'.basename($selfUrl).'?action=open_link&link=', $edit_link.$delete_link.$delete_all.'</li>'],
```
Comment (jmontoyaa) [inline main/admin/configure_homepage.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 170 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260&bid=17216516)

File: main/admin/configure_homepage.php [hunk 3]
```diff
@@ -1207,9 +1242,13 @@ class="form-control"><?php echo $notice_text; ?></textarea>
                         if (!empty($enreg)) {
                             $edit_link = ' <a href="'.$selfUrl.'?action=edit_tabs&amp;link_index='.$tab_counter.'" ><span>'.Display::return_icon('edit.png', get_lang('Edit')).'</span></a>';
                             $delete_link = ' <a href="'.$selfUrl.'?action=delete_tabs&amp;link_index='.$tab_counter.'"  onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete')).'</span></a>';
+                            $delete_all = ' <a href="'.$selfUrl.'?action=delete_all&amp;link_index='.$tab_counter.'"  
+                                            onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) 
+                                            return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete All')).'</span></a>';
```
Comment (jmontoyaa) [inline main/admin/configure_homepage.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 140 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260)

File: main/admin/configure_homepage.php [hunk 4]
```diff
@@ -1207,9 +1242,13 @@ class="form-control"><?php echo $notice_text; ?></textarea>
                         if (!empty($enreg)) {
                             $edit_link = ' <a href="'.$selfUrl.'?action=edit_tabs&amp;link_index='.$tab_counter.'" ><span>'.Display::return_icon('edit.png', get_lang('Edit')).'</span></a>';
                             $delete_link = ' <a href="'.$selfUrl.'?action=delete_tabs&amp;link_index='.$tab_counter.'"  onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete')).'</span></a>';
+                            $delete_all = ' <a href="'.$selfUrl.'?action=delete_all&amp;link_index='.$tab_counter.'"  
+                                            onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\'))
```
Comment (jmontoyaa) [inline main/admin/configure_homepage.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 159 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260)

File: main/admin/configure_homepage.php [hunk 5]
```diff
@@ -1207,9 +1242,13 @@ class="form-control"><?php echo $notice_text; ?></textarea>
...
                             $delete_link = ' <a href="'.$selfUrl.'?action=delete_tabs&amp;link_index='.$tab_counter.'"  onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete')).'</span></a>';
+                            $delete_all = ' <a href="'.$selfUrl.'?action=delete_all&amp;link_index='.$tab_counter.'"  
+                                            onclick="javascript: if(!confirm(\''.addslashes(api_htmlentities(get_lang('ConfirmYourChoice'), ENT_QUOTES)).'\')) 
+                                            return false;"><span>'.Display::return_icon('delete.png', get_lang('Delete All')).'</span></a>';
                             $tab_string = str_replace(
                                 ['href="'.api_get_path(WEB_PATH).'index.php?include=', '</li>'],
-                                ['href="'.api_get_path(WEB_CODE_PATH).'admin/'.basename($selfUrl).'?action=open_link&link=', $edit_link.$delete_link.'</li>'],
+                                ['href="'.api_get_path(WEB_CODE_PATH).'admin/'.basename($selfUrl).'?action=open_link&link=',
```
Comment (jmontoyaa) [inline main/admin/configure_homepage.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 125 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5254260)

**Comment by ywarnier** [comment]
Merged. Thanks and sorry for the delay!

**Pull request merged**